packaging/os/apt_repository.py

```diff
@@ -81,6 +87,12 @@
 # On Ubuntu target: add nginx stable repository from PPA and install its signing key.
 # On Debian target: adding PPA is not available, so it will fail immediately.
 apt_repository: repo='ppa:nginx/stable'
+
+# Call the apt-get update to get all recent package lists with their dependencies.
+apt_repository: repo='deb http://archive.canonical.com/ubuntu hardy partner' update_cache=yes
```

Comment:
Having `update_cache=yes` here misleads the reader into thinking that update_cache defaults to "no".